src/core/components/radio/stories/radio-group/optional.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import { RadioGroup, Radio } from "../../index"
+
+const optional = () => (
+	<form onSubmit={e => e.preventDefault()}>
+		<RadioGroup name="yes-or-no" optional={true}>
```

Comment:
It would be useful if we passed a `label` prop here to make the story feel more realistic and to balance the "Optional" label. Something like: "Do you live in the United Kingdom?"